packages/react/index.js

```diff
@@ -8,3 +8,4 @@
  */
 
 export * from './src/React';
+export {default} from './src/React';
```

Comment:
Isn't `export *` going to be sufficient to cover default too?

Reply:
Nope. That was a surprise to me too.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/export

Reply:
Haha okay, TIL.